crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -474,4 +487,21 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    #[cfg(not(feature = "network-sharding"))]
+    fn test_config_default_network_config_no_sharding() {
```

Comment:
Given the comment above, these would probably need to be a higher level test that starts Kitsune with custom config?

I do like the feature layout on these though, we just need to make CI check both